Expecto/Expecto.Impl.fs

```diff
@@ -494,6 +494,8 @@ module Impl =
       /// An optional filter function. Useful if you only would
       /// like to run a subset of all the tests defined in your assembly.
       filter   : Test -> Test
+      /// List tests of specified state
```

Comment:
What does this mean? Is this a configuration flag?

Reply:
So this is a configuration flag to show only all tests that have one or two or three of the specified states?

Reply:
the proposed usage would be:

- `--list-tests` - lists all tests as before
- `--list-tests focused` - lists focused tests only
- `--list-tests pending focused` - lists pending and focused tests

but I'm of course open for any alternatives.